src/components/portfolioPage/Portfolio.jsx

```diff
@@ -66,8 +81,13 @@ class Portfolio extends Component {
       let intervalTime=Math.max(10000,elapsedTime*5);
       console.log("reload every ",intervalTime);
       self._timer = setInterval(
-        () => {
-          self._fetchAll();
+        async () => {
+          var latest_seq = await this._get_latest_update_seq();
+
+          if(self._get_int_update_seq(self.update_seq)<self._get_int_update_seq(latest_seq)){
```

Comment:
- Being different is enough (how could we go back to the past?).
- Be careful with eslint rules (there should be a space after `if` and spaces around binary operators such as `<` or `==`).